src/coord/cartesian/AxisModel.ts

```diff
@@ -35,6 +36,10 @@ interface CartesianAxisOption extends AxisBaseOption {
     position?: CartesianAxisPosition;
     // Offset is for multiple axis on the same position.
     offset?: number;
+    sort?: boolean;
+    realtimeSort?: boolean;
+    sortSeriesIndex?: number;
```

Comment:
Still not very show if it's better to put `sort` `realtimeSort` configuration on the series. Perhaps we can let developers do the decision.

Reply:
Leave this as it is for now.